app/schemas/models/cinematic.schema.js

```diff
@@ -1,125 +1,99 @@
-  backgroundArt: c.object({
-    title: 'Background Art',
-    description: 'The art in the background of this shot.'
-  }, {
-    // TODO: Will rasterized thangTypes work for backgrounds?
-    type: c.shortString({ enum: ['slug'] }),
-    slug: c.shortString({
-      title: 'Background Slug',
-      description: 'The thangType slug of the background asset'
-    })
-  })
-  // TODO: music
+  backgroundArt: ThangTypeSchema('Background Art', 'The rasterized image to place on the background')
 })
 
 const DialogNode = c.object({
   title: 'Dialog Node',
   description: 'A node of a shot. Contains dialog instructions.',
-  required: ['dialogClear']
 }, {
   speaker: c.shortString({ enum: ['left', 'right'], title: 'Speaker', description: 'Which character is speaking. Used to select speech bubble.' }),
   text: { type: 'string', title: 'Text', description: 'html text', maxLength: 500 },
   i18n: { type: 'object', format: 'i18n', props: ['text'], description: 'Help translate this cinematic dialogNode.' },
   textLocation: c.object({ title: 'Text Location', description: 'An {x, y} coordinate point.', format: 'point2d', required: ['x', 'y'] }, {
     x: { title: 'x', description: 'The x coordinate.', type: 'number', 'default': 0 },
     y: { title: 'y', description: 'The y coordinate.', type: 'number', 'default': 0 } }),
-  action: c.shortString({ title: 'Action', description: 'The action or animation to play on the speaker.' }),
   triggers: c.object({
     title: 'Triggers',
     description: 'Events that can occur during the dialogue.'
   }, {
-    changeBackground: c.object({
-      title: 'Change Background',
-      description: 'Change the background image of the cinematic',
-      required: ['art', 'triggerStart']
-    }, {
-      art: c.shortString({ title: 'Art', description: 'The background art path' }),
-      triggerStart: c.int({ title: 'Trigger Start(ms)', description: 'The number of milliseconds until the background changes.' })
-    }),
     backgroundObject: c.object({
       title: 'Background Object',
       description: 'Add a background object after given duration',
-      required: ['art', 'triggerStart']
+      required: ['thangType', 'triggerStart']
     }, {
-      thangType: c.shortString({ title: 'Art', description: 'The background image ThangType slug.' }),
+      thangType: ThangTypeSchema('Background Object', 'The image to place'),
       triggerStart: c.int({ title: 'Trigger Start(ms)', description: 'The number of milliseconds until background image art appears' })
     }),
-    clearObjects: c.object({
-      title: 'Clear Object',
+    clearBackgroundObject: c.object({
+      title: 'Clear Background Object',
       description: 'Clears the background objects from the screen after a given duration'
     }, {
       triggerStart: c.int({ title: 'Trigger Start(ms)', description: 'The number of milliseconds until background object is cleared' })
-    }),
-    animationTrigger: c.object({
```

Comment:
Why are we removing a bunch of properties from the schema?

Reply:
We add them back later. They were just adding bloat for this PR.